dev-libs/device-atlas-api-c/device-atlas-api-c-2.1.2_p1.ebuild

```diff
@@ -33,6 +33,12 @@ pkg_nofetch() {
 	eerror "Then place the file in ${DISTDIR}/${MY_P}.zip"
 }
 
+src_unpack() {
```

Comment:
This is still wrong and entirely unnecessary. The default implementation (which happens when you do not declare anything) should work here just fine.